slurm/install/install.py

```diff
@@ -507,6 +509,28 @@ def set_hostname(s: InstallSettings) -> None:
     ilib.set_hostname(
         new_hostname, s.platform_family, s.ensure_waagent_monitor_hostname
     )
+    if _is_at_least_ubuntu22() and s.ubuntu22_waagent_fix:
```

Comment:
Is it possible to leave a log message here? About restarting waagent. During Troubleshooting via logs it would be good to know at what point during the install the agent was restarted,